SECURITY.md

```diff
@@ -0,0 +1,31 @@
+# Security Policy
+
+## Supported Versions
+
+NLnet Labs adheres to the straightforward, semantic versioning scheme that is
+commonly used in the software industry.
```

Comment:
If we want to do this properly, we should probably set up `cargo-semver-checks` at some point.